release-1.9/release-1.9.md

```diff
@@ -0,0 +1,89 @@
+# Proposed timeline for 1.9
+We will begin the 1.9 release the first week of October 2017.
```

Comment:
We still don't have a release team selected before the release cycle will start.